sdk/src/main/java/com/microsoft/hydralab/performance/PerformanceTestResult.java

```diff
@@ -17,4 +17,5 @@ public class PerformanceTestResult {
      */
     public List<PerformanceInspectionResult> performanceInspectionResults = new ArrayList<>();
     public String inspector;
+    public String parser;
```

Comment:
Can we turn this into a `enum`? I think this has to be one of the strings declared in `PerformanceResultParser`, using string may cause some subtle bugs if someone makes a typo in parser and it is hard to debug the type unless in runtime.

Reply:
So does `inspector`.